.github/workflows/vm-pr.yml

```diff
@@ -98,15 +131,52 @@ jobs:
         
         # Tests were splitted with --dir and --excludeDir to balance execution times below the 9min mark.
         args: [
-          # '--fork=Berlin --expected-test-amount=33',
           '--fork=Istanbul --dir=GeneralStateTests/stTimeConsuming --expected-test-amount=15561',
-          '--fork=Istanbul --excludeDir=stTimeConsuming --expected-test-amount=19817',
-          # '--fork=Constantinople --dir=GeneralStateTests/stTimeConsuming --expected-test-amount=15561',
-          # '--fork=Constantinople --excludeDir=stTimeConsuming --expected-test-amount=17193',
-          # '--fork=Petersburg --dir=GeneralStateTests/stTimeConsuming --expected-test-amount=15561',
-          # '--fork=Petersburg --excludeDir=stTimeConsuming --expected-test-amount=17174',
```

Comment:
did you mean to keep these commented out here, or should they be removed?

Reply:
@ryanio they are OK to be removed. In fact, they were just moved to these lines: https://github.com/ethereumjs/ethereumjs-vm/pull/951/files/c13ce861a06ce6969e59ede971db755dc2490734#diff-8ac46f60fffb24cf4daf7b192aee8bcd313d6fba95b005aa644e40600739a713R173-R179